quickstarts/microsoft.appplatform/microservice-apps-enterprise-plan/README.md

```diff
@@ -4,24 +4,24 @@ page_type: sample
 products:
 - azure
 - azure-resource-manager
-urlFragment: azure-spring-cloud
+urlFragment: azure-spring-apps
 languages:
 - json
 ---
 # Deploy a simple Azure Spring Apps microservice application
 
-![Azure Public Test Date](https://azurequickstartsservice.blob.core.windows.net/badges/quickstarts/microsoft.appplatform/azure-spring-cloud/PublicLastTestDate.svg)
-![Azure Public Test Result](https://azurequickstartsservice.blob.core.windows.net/badges/quickstarts/microsoft.appplatform/azure-spring-cloud/PublicDeployment.svg)
+![Azure Public Test Date](https://azurequickstartsservice.blob.core.windows.net/badges/quickstarts/microsoft.appplatform/microservice-apps-enterprise-plan/PublicLastTestDate.svg)
```

Comment:
Would it be better if we call this folder microsoft.appplatform/azure-spring-apps/?

Reply:
It's a common name, and we should add more quickstart template here.